download/alpha.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: base
+title: Alpha Download
+navgroup: download
+redirect_from:
+  - /alpha/
+  - /Alpha/
```

Comment:
we don't need these redirects for new pages (they were only there for not breaking the old MediaWiki urls on mono-project.com)